test/cadet_web/controllers/user_controller_test.exs

```diff
@@ -60,13 +65,18 @@ defmodule CadetWeb.UserControllerTest do
       |> Assessment.changeset(%{story: nil})
       |> Repo.update()
 
-      story =
+      resp_story =
         conn
         |> get("/v1/user")
         |> json_response(200)
         |> Map.get("story")
 
-      assert story == Enum.fetch!(assessments, 1).story
+      expected_story = %{
+        "story" => Enum.fetch!(assessments, 1).story,
```

Comment:
Ehh, `assessments` is a tuple right? Then I think you should pattern match on it instead.

Reply:
It's a list and i'm getting the second item so I feel it actually looks worse:

```
      [_, assessment | _] = assessments

      expected_story = %{
        "story" => assessment.story,
        "allAttempted" => false
      }
```

Reply:
Oops I derped

Reply:
Oh shoot sorry I derped